projects/1.20-fabric/assets/more-wizards/morewizardsmod/lang/zh_cn.json

```
@@ -0,0 +1,41 @@
+{
+  "item.morewizardsmod.more_wizards_lightning_spell_book": "雷电魔法书",
+  "item.morewizardsmod.more_wizards_soul_spell_book": "灵魂魔法书",
+  "item.morewizardsmod.lightning_staff": "雷电权杖",
+  "item.morewizardsmod.netherite_lightning_staff": "下界合金雷电权杖",
+  "item.morewizardsmod.soul_staff": "灵魂权杖",
+  "item.morewizardsmod.netherite_soul_staff": "下界合金灵魂权杖",
+
+  "item.morewizardsmod.lightning_wizard_armor_t1_head": "雷电学徒帽",
+  "item.morewizardsmod.lightning_wizard_armor_t1_chest": "雷电学徒长袍 ",
+  "item.morewizardsmod.lightning_wizard_armor_t1_legs": "雷电学徒护腿",
+  "item.morewizardsmod.lightning_wizard_armor_t1_feet": "雷电学徒靴",

```

```suggestion
  "item.morewizardsmod.lightning_wizard_armor_t1_feet": "雷电学徒靴子",
```
原版的固有名词应与Minecraft原版一致，如有特殊用法请指出。
具体可见我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **06.专有名词** 一节。
你也可以通过上方评论中，bot生成的对比（diff）文件或 [Minecraft 译名标准化](https://zh.minecraft.wiki/w/Minecraft_Wiki:%E8%AF%91%E5%90%8D%E6%A0%87%E5%87%86%E5%8C%96) 以获取更多的原版翻译帮助。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。